README.md

```diff
@@ -1,10 +1,189 @@
-# criteo-java-marketing-sdk
+# Criteo Marketing SDK for Java
```

Comment:
This README is nice, it would be better to merge https://github.com/criteo/criteo-marketing-sdk-generator/pull/13 first ;)